src/Trigger/RabbitMQListener.cs

```diff
@@ -139,12 +144,77 @@ internal void RepublishMessages(BasicDeliverEventArgs ea)
             }
         }
 
+        public string Id
+        {
+            get
+            {
+                return $"RabbitMQTrigger-{_queueName}".ToLower();

```

Comment:
Recommend adding the FunctionId to this as well. Take a look, at the QueueListener example - it uses {_functionId}-QueueTrigger-{_queue.Name}. FuncitonId represents the individual function the listener is for.